### PR TITLE
[opensuse] updated whitelistings for gdm, systemd_loadkey and whitelisting maintenance

### PR DESCRIPTION
**Pull request by mgerstner**

### CODE REVIEW

File: configs/openSUSE/dbus-services.toml
Comment (FilippoBonazziSUSE) [inline configs/openSUSE/dbus-services.toml:1]: Ok

File: configs/openSUSE/pam-modules.toml
Comment (FilippoBonazziSUSE) [inline configs/openSUSE/pam-modules.toml:1]: Ok

**Review by FilippoBonazziSUSE — APPROVED** [review]
All ok

**Comment by FilippoBonazziSUSE** [comment]
@mgerstner it looks like we jumped the gun with the GDM whitelisting, perhaps we should have kept both old and new for a while (see https://build.opensuse.org/request/show/1155534#comment-1905827). Can you look into this?

**Comment by mgerstner** [comment]
> @mgerstner it looks like we jumped the gun with the GDM whitelisting, perhaps we should have kept both old and new for a while (see https://build.opensuse.org/request/show/1155534#comment-1905827). Can you look into this?

Usually the new packages go directly into Factory and that's it. If there is a problem in this case then it is no problem to re-create the old entry. Even you can do it :-)

**Pull request merged**